teams/windows.toml

```diff
@@ -0,0 +1,8 @@
+name = "windows"
+marker-team = true
+
+[people]
+leads = []
+members = [
+    "spastorino",
```

Comment:
Can you add the following people?
* @rylev 
* @danielframpton
* @sivadeilra 
* @lzybkr 
* @kennykerr
* @gdr-at-ms
* @luqmana
* @arlosi

Reply:
And me!

Reply:
Done!.